doc/transform.ipynb

```diff
@@ -0,0 +1,7402 @@
+       "      dtype=float32)</pre></li><li class='xr-var-item'><div class='xr-var-name'><span class='xr-has-index'>lat_u</span></div><div class='xr-var-dims'>(lat_u)</div><div class='xr-var-dtype'>float32</div><div class='xr-var-preview xr-preview'>-74.0 -73.66667 ... 64.66566</div><input id='attrs-2c23ee0a-fcd9-4a57-a390-f57501c9e409' class='xr-var-attrs-in' type='checkbox' ><label for='attrs-2c23ee0a-fcd9-4a57-a390-f57501c9e409' title='Show/Hide attributes'><svg class='icon xr-icon-file-text2'><use xlink:href='#icon-file-text2'></use></svg></label><input id='data-726b0438-dda4-4440-991d-6316c5587517' class='xr-var-data-in' type='checkbox'><label for='data-726b0438-dda4-4440-991d-6316c5587517' title='Show/Hide data repr'><svg class='icon xr-icon-database'><use xlink:href='#icon-database'></use></svg></label><div class='xr-var-attrs'><dl class='xr-attrs'><dt><span>GridType :</span></dt><dd>Cylindrical Equidistant Projection Grid</dd><dt><span>actual_range :</span></dt><dd>[-74.0, 64.9990005493164]</dd><dt><span>axis :</span></dt><dd>Y</dd><dt><span>long_name :</span></dt><dd>latitude</dd><dt><span>standard_name :</span></dt><dd>latitude</dd><dt><span>units :</span></dt><dd>degrees_north</dd></dl></div><pre class='xr-var-data'>array([-74.     , -73.66667, -73.33334, ...,  63.99901,  64.33234,  64.66566],\n",
+       "      dtype=float32)</pre></li><li class='xr-var-item'><div class='xr-var-name'><span class='xr-has-index'>level</span></div><div class='xr-var-dims'>(level)</div><div class='xr-var-dtype'>float32</div><div class='xr-var-preview xr-preview'>5.0 15.0 25.0 ... 3972.0 4478.0</div><input id='attrs-b828bb47-b753-40f5-884d-c7ed67cccf13' class='xr-var-attrs-in' type='checkbox' ><label for='attrs-b828bb47-b753-40f5-884d-c7ed67cccf13' title='Show/Hide attributes'><svg class='icon xr-icon-file-text2'><use xlink:href='#icon-file-text2'></use></svg></label><input id='data-a47bfbf5-0c1f-4541-a87a-c5c6e46245e9' class='xr-var-data-in' type='checkbox'><label for='data-a47bfbf5-0c1f-4541-a87a-c5c6e46245e9' title='Show/Hide data repr'><svg class='icon xr-icon-database'><use xlink:href='#icon-database'></use></svg></label><div class='xr-var-attrs'><dl class='xr-attrs'><dt><span>actual_range :</span></dt><dd>[5.0, 4478.0]</dd><dt><span>axis :</span></dt><dd>Z</dd><dt><span>long_name :</span></dt><dd>depth below sea level</dd><dt><span>positive :</span></dt><dd>down</dd><dt><span>units :</span></dt><dd>m</dd></dl></div><pre class='xr-var-data'>array([   5.,   15.,   25.,   35.,   45.,   55.,   65.,   75., 
```

Comment:
This should be something like `Grid.guess_coord(axis="Z", coord="outer", name="level_outer")` for convenience

Reply:
We do have the auto generate module, which "should" be able to do that. It requires some overhauling though!